package-defaults.yaml

```diff
@@ -11,6 +11,7 @@ default-extensions:
 - ConstraintKinds
 - DataKinds
 - DefaultSignatures
+- DerivingStrategies
```

Comment:
This allows us to be more explicit about where our instances should come from; see the above `deriving newtype (...)`

Reply:
Heh!  You found an excuse for turning that on.  :-)  :+1: